--- rtl8188EUS_linux_v4.3.0.8_13968.20150417/include/rtw_debug.h	2015-04-17 03:54:43.000000000 -0300
+++ rtl8188EUS_linux_v4.3.0.8_13968.20150417/include/rtw_debug.h	2016-01-24 04:25:34.370994816 -0300
@@ -184,7 +184,7 @@ extern void rtl871x_cedbg(const char *fm
 	#define _seqdump(sel, fmt, arg...) _dbgdump(fmt, ##arg)
 #elif defined PLATFORM_LINUX
 	#define _dbgdump printk
-	#define _seqdump seq_printf
+	#define _seqdump(sel, fmt, arg...) _dbgdump(fmt, ##arg)
 #elif defined PLATFORM_FREEBSD
 	#define _dbgdump printf
 	#define _seqdump(sel, fmt, arg...) _dbgdump(fmt, ##arg)
