--- rtl8188EUS_linux_v4.3.0.7_12758.20141114/Makefile	2014-11-14 06:28:47.000000000 -0300
+++ rtl8188EUS_linux_v4.3.0.7_12758.20141114/Makefile	2015-02-05 23:47:06.181672235 -0300
@@ -7,6 +7,7 @@ EXTRA_CFLAGS += -O1
 #EXTRA_CFLAGS += -pedantic
 #EXTRA_CFLAGS += -Wshadow -Wpointer-arith -Wcast-qual -Wstrict-prototypes -Wmissing-prototypes
 
+EXTRA_CFLAGS += -Wno-error=date-time
 EXTRA_CFLAGS += -Wno-unused-variable
 EXTRA_CFLAGS += -Wno-unused-value
 EXTRA_CFLAGS += -Wno-unused-label
